src/drawing/__tests__/DrawingAnnotations-test.tsx

```diff
@@ -232,4 +232,28 @@ describe('DrawingAnnotations', () => {
             },
         );
     });
+
+    describe('state changes', () => {
+        test.each`
+            drawn         | initialDrawn | numberOfCalls
+            ${pathGroups} | ${[]}        | ${2}
+            ${[]}         | ${[]}        | ${0}
```

Comment:
Should we test that passing the same `pathGroups` object also does not cause an update?

Reply:
Yes we do.